src/cljs/main/broadfcui/auth.cljs

```diff
@@ -9,6 +9,7 @@
    [broadfcui.config :as config]
    [broadfcui.nav :as nav]
    [broadfcui.utils :as utils]
```

Comment:
It's a bit confusing that we still have a `utils` file that's a sibling to the `utils` directory, but maybe you have further plans for this

Reply:
It's the same kind of pattern that we have for table.